### ISSUE TITLE
Experiment results

### ISSUE THREAD

**Issue by IbsenChan** [WenbinLee/DN4#9]
Hi,

Thank you very much to share your code with us!

I try to reproduce your results in your paper, and rerun you code directly. But I found the results always a bit lower than that reported in your paper. Should some hyperparameters be tuned?

Also，I found the learning rate decayed 0.05 each interval. Should it be changed to 0.5, or 0.05 is OK?

Hope to hear from you~

**Comment by WenbinLee** [comment]
It may be caused by the different initializations of the neural network by different versions of Pytorch. If you find it is a little overfitting, you can use a smaller initial learning rate instead of 0.005, e.g., 0.001. Actually, we had tested the released code on the miniImageNet dataset in our own machine and you can find our tested results in the file of DN4/results. By the way, can you tell me the version of Pytorch you used and the results you obtained? I can retest the code on other Pytorch versions and machines. 

It is a little trick. We didn't pay a lot of effort on this hyperparameter. In our experiments, we find 0.05 is okay because we use 0.005 as the initial learning rate for the first 10 epoch. Maybe you can use 0.001 as the initial learning rate, and change it by 0.5. We believe we should have a better decay mechanism of the learning rate, but this is not the main concern of our paper. Thank you.

**Comment by happycoding1996** [comment]
@IbsenChan Could you share the results you have reproduced with this code? Thank you!
